web/src/components/shared/saleComponent/SaleComponent.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { SaleItem } from '../saleItem/SaleItem';
+import './style.css';
+
+const SaleComponent = () => {
+    return (
+        <div className="sale__container">
+            <img
+                className="sale__container__image"
+                src="https://d-art.ppstatic.pl/kadry/k/r/1/9d/0a/6234b5b877108_o_full.jpg"
+                alt="photo"
+            ></img>
+
+            <div className="sale__container__items">
+                <SaleItem
+                    name="New Balance"
+                    img="https://www.eobuwie.com.pl/media/catalog/product/cache/image/650x650/0/1/01_0000300270608_is.jpg"
+                    price={279}
+                />
+                <SaleItem
+                    name="New Balance"
+                    img="https://www.eobuwie.com.pl/media/catalog/product/cache/image/650x650/0/1/01_0000300270608_is.jpg"
+                    price={279}
+                />
+                <SaleItem
+                    name="New Balance"
+                    img="https://www.eobuwie.com.pl/media/catalog/product/cache/image/650x650/0/1/01_0000300270608_is.jpg"
+                    price={279}
+                />
+
+                <SaleItem
+                    name="New Balance"
+                    img="https://www.eobuwie.com.pl/media/catalog/product/cache/image/650x650/0/1/01_0000300270608_is.jpg"
+                    price={279}
+                />
+
+                <SaleItem
+                    name="New Balance"
+                    img="https://www.eobuwie.com.pl/media/catalog/product/cache/image/650x650/0/1/01_0000300270608_is.jpg"
+                    price={279}
+                />
+
+                <SaleItem
+                    name="New Balance"
+                    img="https://www.eobuwie.com.pl/media/catalog/product/cache/image/650x650/0/1/01_0000300270608_is.jpg"
+                    price={279}
+                />
```

Comment:
whats that? We dont hardcode things anymore
get products from context. add to the first and second one some start points. display products that have only 2 or more stars.